buildstockbatch/gcp/gcp.py

```diff
@@ -496,6 +495,41 @@ def get_existing_batch_job(self):
         except exceptions.NotFound:
             return None
 
+    def get_existing_postprocessing_job(self):
+        jobs_client = run_v2.JobsClient()
+        try:
+            job = jobs_client.get_job(name=self.postprocessing_job_name)
+            return job
+        except exceptions.NotFound:
+            return False
+
+    def check_for_existing_jobs(self, pp_only=False):
+        """If there are existing jobs with the same ID as this job, logs them as errors and returns True.
+
+        Otherwise, returns False.
```

Comment:
nit: unnecessary, imho. but if you want the clarity, put in the `:return:` part of the doc